🚀[Release v3.12.0] Merge into Main

kcw-grunt opened the pull request:
# What's New
- Added a address test
- New Email and notification signups 
- Other improvements
- Minor layout fixes

kcw-grunt commented:
cc: @zenithquinn this is a result of some user feedback.

mosadialiou commented on loafwallet/RegistrationViewModel.swift:
```diff
@@ -172,7 +172,8 @@ class RegistrationViewModel: ObservableObject {
         
-        guard (passwordString.count >= 6 && passwordString.count <= 10) else {
+        //MARK: -  Max Password 24 chars
+        guard (passwordString.count >= 6 && passwordString.count <= 24) else {
```
@kcw-grunt I don't know what was the reason to limit the password to 10 or even 24 chars. I do think we just have to check the min number of chars. If a user want he can create a 50 chars password length.

kcw-grunt replied:
yeah...the partner are not giving details....I
'll take that line out.

mosadialiou commented on loafwallet/RegistrationViewModel.swift:
```diff
@@ -170,11 +170,7 @@ class RegistrationViewModel: ObservableObject {
             return false
-        }
-        
-        guard (passwordString.count >= 6 && passwordString.count <= 10) else {
```
@kcw-grunt The maximum number of characters is not needed but I do think we should keep the minimum characters at 6. Thoughts?

kcw-grunt replied:
The Ternio team haven't provided a schema yet. Standby on this. I will update the PR soon.

mosadialiou replied:
ok

mosadialiou reviewed (approved):
LGTM

kcw-grunt committed:
🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

kcw-grunt committed:
Release/v3.9.0 (#128)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump

kcw-grunt committed:
🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

kcw-grunt committed:
buiid bump (#125)

kcw-grunt committed:
reemvoed conflict tags

kcw-grunt committed:
🚀[Release v3.10.1] Merge into Develop (#162)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🚧[Techdebt] Firebase polish 121 (#139)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* Create sync-mirror.yml

* Changed the clone sig

* Update sync-mirror.yml

- added sync branches and tags

* Update sync-mirror.yml

refactor

* Update sync-mirror.yml

fixed uname

* Update sync-mirror.yml

* Update sync-mirror.yml

* Create PubtoPrivate.yml

* Update sync-mirror.yml

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* build bump

-clean up strings file
- Udpated to iOS 14.0 as a target
https://www.statista.com/statistics/1118925/mobile-apple-ios-version-share-worldwide/

* Removed cruft

* Updated the Firebase constants

- changed it to lowercase

* updated links

* fixed constants

-build bump

* Mark unused code (#147)

* Added and modified a ruby script that shows a warining if a class, function or variable is not used

* Fixed static function partnersDataArray

* Updated build number

* build bump

* clean up duplicate scripts

* readded "Send.UnstoppableDomains.udSystemError" = "System lookup problem. [Error: %2$d]";

-bumped the version and build to test Bitrise

* fixed today extension version and build number

* fixed coding norm (space after colon)

- fixed Turkish - Italian mixup
- version bump

Iferencak committed:
Add SwiftFormat to the codebase (#159)

* Added SwiftFormat to the run scripts

* Removed --elseOnSameLine from rules as it is on by default

* Added --disable enumNamespaces rule

* Using SwiftFormat to auto-format current code

* Removed --allman rule

* 🚀[Release v3.10.1] Merge into Develop (#162)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🚧[Techdebt] Firebase polish 121 (#139)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* Create sync-mirror.yml

* Changed the clone sig

* Update sync-mirror.yml

- added sync branches and tags

* Update sync-mirror.yml

refactor

* Update sync-mirror.yml

fixed uname

* Update sync-mirror.yml

* Update sync-mirror.yml

* Create PubtoPrivate.yml

* Update sync-mirror.yml

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* build bump

-clean up strings file
- Udpated to iOS 14.0 as a target
https://www.statista.com/statistics/1118925/mobile-apple-ios-version-share-worldwide/

* Removed cruft

* Updated the Firebase constants

- changed it to lowercase

* updated links

* fixed constants

-build bump

* Mark unused code (#147)

* Added and modified a ruby script that shows a warining if a class, function or variable is not used

* Fixed static function partnersDataArray

* Updated build number

* build bump

* clean up duplicate scripts

* readded "Send.UnstoppableDomains.udSystemError" = "System lookup problem. [Error: %2$d]";

-bumped the version and build to test Bitrise

* fixed today extension version and build number

* fixed coding norm (space after colon)

- fixed Turkish - Italian mixup
- version bump

* Added SwiftFormat to the run scripts

* Removed --elseOnSameLine from rules as it is on by default

* Added --disable enumNamespaces rule

* Using SwiftFormat to auto-format current code

* Removed --allman rule

* PartnerData auto-format

* Added marking of unused code back to runscripts

* Delete PubtoPrivate.yml

* Delete sync-mirror.yml

Co-authored-by: Kerry Washington <mrkerrywashington@icloud.com>

kcw-grunt committed:
🤬[Techdebt] #167 add custom user agent++ (#168)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* Create sync-mirror.yml

* Changed the clone sig

* Update sync-mirror.yml

- added sync branches and tags

* Update sync-mirror.yml

refactor

* Update sync-mirror.yml

fixed uname

* Update sync-mirror.yml

* Update sync-mirror.yml

* Create PubtoPrivate.yml

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🚧[Techdebt] Firebase polish 121 (#139)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* Create sync-mirror.yml

* Changed the clone sig

* Update sync-mirror.yml

- added sync branches and tags

* Update sync-mirror.yml

refactor

* Update sync-mirror.yml

fixed uname

* Update sync-mirror.yml

* Update sync-mirror.yml

* Create PubtoPrivate.yml

* Update sync-mirror.yml

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* build bump

-clean up strings file
- Udpated to iOS 14.0 as a target
https://www.statista.com/statistics/1118925/mobile-apple-ios-version-share-worldwide/

* Removed cruft

* Updated the Firebase constants

- changed it to lowercase

* updated links

* fixed constants

-build bump

* Mark unused code (#147)

* Added and modified a ruby script that shows a warining if a class, function or variable is not used

* Fixed static function partnersDataArray

* Updated build number

* changed the urtl

- change the deployment target

* removed staging server url

normalizing to the prod server only

* aded small test

- build bump

* Retired old https server code (#154)

* Updated

* 🚀[Release v3.10.1] Merge into Develop (#162)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🚧[Techdebt] Firebase polish 121 (#139)

* 🚀[Release v3.10.0] Merge into Main  (#136)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* Create sync-mirror.yml

* Changed the clone sig

* Update sync-mirror.yml

- added sync branches and tags

* Update sync-mirror.yml

refactor

* Update sync-mirror.yml

fixed uname

* Update sync-mirror.yml

* Update sync-mirror.yml

* Create PubtoPrivate.yml

* Update sync-mirror.yml

* 🚀[Release v3.10.0] Merge into Develop (#135)

* 🦺[Techdebt] fix firebase (#115)

* Update firebase config
 auto clean up strings

- version bump
- Commented out debug FB file

* Checked the version is sending FB data

- bump build number

* 🥳[Techdebt] relayout address view controller (#120)

* Successfully placed in UIHosting Address

Updated the strings file

* Added event

* added delay

- used for animating U Domain

* Added function of subviews in SendViewController

Layout is updated

* Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

* Refactored SendButton

- Using UIHostingViewController

* Successfully send LTC

-WIP: fix white space

* Refactored the layout of the send  button

* buiid bump (#125)

* [Techdebt] ci cd refactor (#132)

* Removed flaky tests

* build bump

* build bump

* removed unused file

- build bump

* build bump

* fixed scan QR (#130)

* fixed scan QR

* update gitignore

* Feature/add bitrefill new (#134)

* Added llocalView

- added bitrefill
- basic function is available
- build bump

* bump

* added basic web widget

* updated logo

* build bump

* changed the Bitrefill link

- build bump

* pre-bump version

- to get a binary to TestFlight

* build bump

* build bump again

* Clean up background

* build bump

* build bump

-clean up strings file
- Udpated to iOS 14.0 as a target
https://www.statista.com/statistics/1118925/mobile-apple-ios-version-share-worldwide/

* Removed cruft

* Updated the Firebase constants

- changed it to lowercase

* updated links

* fixed constants

-build bump

* Mark unused code (#147)

* Added and modified a ruby script that shows a warining if a class, function or variable is not used

* Fixed static function partnersDataArray

* Updated build number

* build bump

* clean up duplicate scripts

* readded "Send.UnstoppableDomains.udSystemError" = "System lookup problem. [Error: %2$d]";

-bumped the version and build to test Bitrise

* fixed today extension version and build number

* fixed coding norm (space after colon)

- fixed Turkish - Italian mixup
- version bump

* added custom var

* build bump

* Add codesigndoc ignore

* Added Swift Packages to the project

- Closes : https://github.com/litecoin-foundation/litewallet-ios/issues/156

* Removed pods

* Added Litewallet PartnerAPI to project

* SUCCESS!

- Added PartnerAPI as a Submodule

* Rebase from SwiftFormat

* Update .gitignore

* Removed submodule

* resetting add the submodule

* updated config

* Removed PartnerAPI #170

Removed references to the LitewalletPartnerAPI lib
- Compiled the app
- Added LitecoinCard View for CardView
- Added CardWebViewController
- Deleted unused views and viewModels

* change build number

* Rmeoved all LitecoinCard Tests

* build bump

* Failed Test for Bitrise CI

* Unbroken tests

* small change build bump

* another build bump

* build again

* build bump

* yet again bump

* small change

* matching build number with bitrise run

* nonsense change

* Fixed the UDSystem Error issues

- removed commented out PartnerAPI Code

Co-authored-by: Iferencak <iferencak@profico.hr>

kcw-grunt committed:
bump to match current prod build number

kcw-grunt committed:
nit…format (#203)

kcw-grunt committed:
Update the loafwallet-core submodule

kcw-grunt committed:
#207 resolve core update issue missing labels (#211)

* move the git submodule to the latest

- v1.0.0 releease of loafwallet-core

# Conflicts:
#	Modules/loafwallet-core

* ISSUE: Not able to type and display recv addresses

- There is a failiure in the  conversion from CChar to Int8
- Have send working
- updated loafwallet-core v1.1.0
- Added falsePositived interface
- Added enum and set values for
 - low privacy
 - semi-private
 - anonymous
- Updated the loafwallet-core
- adds the fpRate var

* Resolved PeerManager set with proper optional setting.

* WIP: Cannot unwrap ltc1 addresses to retain them in the UILabel

- ltc1 addresses are present but not unwrapping properly
@Iferencak this is the disappearing error you saw when we updated the loafwallet-core.
- Not able to figure this out yet.
- All ideas are welcome

* WIP:  Still debugging

- The issue is the BRCore methods compare the `ltc1` address against the set of addresses and it fails because the BRSet is not setup for the address.

The issue is this worked before and I need to investigate what happened before and how things changed

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* move the git submodule to the latest

- v1.0.0 releease of loafwallet-core

* Bugfix!: Duplicated char in converter

Added optic enum

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Move all the extensions from BRCore to separate files

- Moved these into separate files to help debug

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Refactored the conversion

- Fixed a bug where a element is duplicated
`unichar(charArray.20),unichar(charArray.21), unichar(charArray.22), unichar(charArray.23), unichar(charArray.23), unichar(charArray.24)`

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

---------

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

kcw-grunt committed:
🚀[Release v3.11.1] Merge into Develop  (#214)

* bump build to date based number

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Removed Ternio Card code

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Polished the version tag in the lock screen

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Build bump

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Removed TabBarVC

- Cleanup to remove old references

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Updated the Infura and partner IPFS access

- Required in the latest UD library

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Added commit to partner keys

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

---------

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

kcw-grunt committed:
Removed all ArticleIds (#216)

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

# Conflicts:
#	loafwallet/ArticleIds.swift

kcw-grunt committed:
updated the gitignore

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

kcw-grunt committed:
😍[Feature] Exploration/update create restore wallet (#219)

* RENAMED the project to Litewallet

- Added new litewallet.entitlements
- Remapped files
- build bumped
- Updated to Swift 4.2
- Version bump
- Added PaddedUILabel (unused
- removed UD
- refactored layouts of SendViewController
- bumped build number

* Updated the fee words for all languages

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

updated the strings’ fees’

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

WIP Adding in the ops fees

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

adjustment of the Send VW  layout

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

Removed ifdef for old versions of iOS

 other refactor

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Added new EnterPhraseView HostingController

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* WIP: Pusher waiting for https://github.com/pusher/push-notifications-swift/issues/187

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Integrating the ops fee

Debugging to find where the amount was set
re-add Satoshis

* Successfully send to 2 outputs
Sent 2 outputs with 1 input

Need to readd refactor
remove cruft
refactored constant
fixed dupe constant
Further polish for confirmation for sending

* Integrated Push Notifications

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Refactored legacy code

- Moved Application Extension funcs to different file
- Added new HostingController

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Refactored the Launch

- Added SwiftUI Font for more widespread use
- Cleaned up  logo

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* WIP: Adding more create views

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Began added CreateViews

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Updated the  localization files

- update the schemes
- Refactor to fix old code / delete unused files
- Added audio that plays when user selects language
- Working!  The word billboard is working
- updated the taglines
- updated the alert message for the language alert
- Good language switching

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Added pusher and paths for devices

- Re-add the partner plist
- Fixes the iPad idiom by giving those users a legacy UIKit path
- refactor unused code
- auto linting
- updated the pusher option

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Finished layout of Acceptance for notifications

propgated language to Accepted notification
auto format

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Fixes a memory leak solved by @hectorchu

- We had strong reference to the observer(s) and this was causing lingering issues causing the app to crash at rutime
- Updated the key file references

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* updated loafwallet-core version

-Updated to loafwallet-core v1.6.1
-update urls
-Signup sheet is working!!!!!
-polished the signup
- Re added some callbacks that were updating the recall big bug was fixed

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Need to work in the Showing and hiding the announce view

Haven’t figured out how to show and hide the AnnounceView then show the tabVC

- Sometimes it reads the evaluation and sometimes it doesnt
- removed the current user signup sheet

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* added LWAnalytics notifications

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* added amount field

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* prelayout of the amount in the send SwiftUI send

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* added memo and Amount

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* added hack to make it work

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* cleanup of unused code

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* cleanup of other efforts to debug

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* readded the didStartEditing

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

* Readded then Address  Cell View

- Still have the issues the ModalViewController height

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

---------

Signed-off-by: kcw-grunt <mrkerrywashington@icloud.com>

kcw-grunt committed:
updated filesystem

josikie reviewed (approved):
Cool! Thank you @kcw-grunt
